### PR TITLE
Introduce ADRs

### PR DESCRIPTION
**Pull request by joschka**
wdyt?

### CODE REVIEW

**Review by pgurusinga — APPROVED** [review]

File: doc/adr/0001-record-any-decisions.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+The _classic_ Nygard template is very visible and well adopted and plenty of [tooling](https://adr.github.io/#decision-capturing-tools) exists. It's also already used in the company by other teams.
+
+The more _modern_ [MADR 3.0.0](https://adr.github.io/madr/) is offered as a template to capture any decisions, as its makers believe that any (important) decision should be captured - not only architectural significant ones. It broadens the scope of ADRs to "Any Decision Records" and advocates to not take the term "architecture" too seriously. It lacks adoption and tooling support.
+
+## Consequences
+
+With [adr-tools](https://github.com/npryce/adr-tools) it's as frictionless as possible to start working on a new ADR. Others can find our ADRs in the expected place `doc/adr` and in the familiar Nygard format.
+
+By defining ADRs as _Any Decision Records_ it will hopefully be less daunting to record a decision. The resulting decision log might be architecturally less terse but that is a trade-off we are willing to accept if it increases the chance that decision logging becomes a habit of the team.
```
Comment (frederike-ramin) [inline doc/adr/0001-record-any-decisions.md:29]: I like this a lot. Definitely tackles an insecurity I have around ADRs 👍

File: doc/adr/0001-record-any-decisions.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+## Decision
+
+We will use Architecture Decision Records, as [described by Michael Nygard](http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions). We extend their usage to include other (important) decisions, hence calling them _Any Decision Records_.
+
+We considered two template options.
+
+The _classic_ Nygard template is very visible and well adopted and plenty of [tooling](https://adr.github.io/#decision-capturing-tools) exists. It's also already used in the company by other teams.
+
+The more _modern_ [MADR 3.0.0](https://adr.github.io/madr/) is offered as a template to capture any decisions, as its makers believe that any (important) decision should be captured - not only architectural significant ones. It broadens the scope of ADRs to "Any Decision Records" and advocates to not take the term "architecture" too seriously. It lacks adoption and tooling support.
```
Comment (frederike-ramin) [inline doc/adr/0001-record-any-decisions.md:23]: I'm a little confused with this. So you introduced both possibilities - like that - but then the actual decision between the two is missing? Right now in the Consequences, it sounds like we're doing a mixture? Using the familiar format but with any decision, not just architectural? I think we can make that more clear here.

**Review by frederike-ramin — COMMENTED** [review]
Glad you're bringing this up again. :)

**Review by frederike-ramin — APPROVED** [review]
From my side you can change the status to "Accepted" ;)

**Pull request merged**